README.md

```diff
@@ -1,3 +1,11 @@
 # AppHealthReport
 
-Module for installing AppHealthReport components
\ No newline at end of file
+The module installs AppHealthReport components:
```

Comment:
The Module doesn't install components. The components are part of the AppHealthReport Module

Reply:
Okay, I got it, I fixed it